bioconda_utils/utils.py

```diff
@@ -251,35 +258,66 @@ def build(recipe, env, verbose=False, testonly=False, force=False):
     """
     print("Building/testing", recipe, "for environment:")
     print(*('\t{}={}'.format(*i) for i in sorted(env)), sep="\n")
-    try:
-        out = None if verbose else sp.PIPE
-        build_args = []
-        if testonly:
-            build_args.append("--test")
-        else:
-            build_args += ["--no-anaconda-upload"]
-        if not force:
-            build_args += ["--skip-existing"]
-        sp.run(["conda", "build", "--quiet", recipe] + build_args,
-               stderr=out,
-               stdout=out,
-               check=True,
-               universal_newlines=True,
-               env=merged_env(env))
+    build_args = []
+    if testonly:
+        build_args.append("--test")
+    else:
+        build_args += ["--no-anaconda-upload"]
+    if not force:
+        build_args += ["--skip-existing"]
+    if docker is not None:
+        conda_build_folder = os.path.join(os.path.dirname(os.path.dirname(shutil.which("conda"))), "conda-bld")
+        recipe = os.path.relpath(recipe, recipe_folder)
+        env = dict(env)
+        env["ABI"] = 4
+        container = docker.create_container(
+            image="continuumio/conda_builder_linux:latest",
```

Comment:
Seems reasonable to pin, and also to somehow propagate that version to the package build metadata somehow so we can keep track.